worlds/dark_souls_3/data/items_data.py

```diff
@@ -404,9 +535,11 @@
     "Jailbreaker's Key",
     "Prisoner Chief's Ashes",
     "Old Cell Key",
-    "Jailer's Key Ring",
+    "Jailer's Key Ring"
 }
 
-item_tables = [weapons_upgrade_5_table, weapons_upgrade_10_table, shields_table, armor_table, rings_table, spells_table, misc_items_table, goods_table]
+item_tables = [weapons_upgrade_5_table, weapons_upgrade_10_table, shields_table,
+               armor_table, rings_table, spells_table, misc_items_table, armor_2_table, goods_table]
```

Comment:
"armor_2_table" should be placed at the end of the array.

```suggestion
               armor_table, rings_table, spells_table, misc_items_table, goods_table, armor_2_table]
```